### PR TITLE
Fix bug in signed_distance for 2D

### PR DESCRIPTION
**Pull request by alecjacobson**
[Describe your changes and what you've already done to test it.]

#### Check all that apply (change to `[x]`)
- [ ] All changes meet [libigl style-guidelines](https://libigl.github.io/style-guidelines/).
- [ ] Adds new .cpp file.
- [ ] Adds corresponding unit test.
- [ ] Adds corresponding python binding.
- [ ] This is a minor change.

### CODE REVIEW

File: tests/include/igl/signed_distance.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      //igl::SIGNED_DISTANCE_TYPE_WINDING_NUMBER,
+      //igl::SIGNED_DISTANCE_TYPE_DEFAULT       ,
+      igl::SIGNED_DISTANCE_TYPE_UNSIGNED      
+      })
+  {
+    Eigen::VectorXd S;
+    Eigen::VectorXi I;
+    Eigen::MatrixXd C,N;
+    igl::signed_distance( P,V,F,type,S,I,C,N);
+    //Eigen::VectorXd Sexact (1,1);Sexact<<sqrt(2.)/2.;
```
Comment (jdumas) [inline tests/include/igl/signed_distance.cpp]: Shouldn't those lines be uncommented?

**Comment by jdumas** [comment]
Linking #1367 to this PR:

File: tests/include/igl/signed_distance.cpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    0,1,
+    1,2,
+    2,0;
+  Eigen::MatrixXd P(1,2);
+  P<<1,1;
+  for(const igl::SignedDistanceType type :
+      {
+      igl::SIGNED_DISTANCE_TYPE_PSEUDONORMAL  ,
+      //igl::SIGNED_DISTANCE_TYPE_WINDING_NUMBER,
+      //igl::SIGNED_DISTANCE_TYPE_DEFAULT       ,
```
Comment (jdumas) [inline tests/include/igl/signed_distance.cpp]: Those two lines are still commented. Can we enable them?

**Comment by alecjacobson** [comment]
Yes w should. I thought I did....

> .
>

**Pull request merged**